src/main/types/SIOP.types.ts

```diff
@@ -50,7 +50,7 @@ export interface RequestRegistrationPayload {
 
 export interface VerifiedAuthenticationRequestWithJWT extends VerifiedJWT {
   payload: AuthenticationRequestPayload; // The unsigned Authentication Request payload
-  presentationDefinition?: PresentationDefinition; // The optional presentation definition object that the RP requests
+  presentationDefinitions?: PresentationDefinitionWithLocation[]; // The optional presentation definition objects that the RP requests
```

Comment:
this should be single

Reply:
No because this is an interpreted field not a payload